interfaces/IWeth.sol

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright 2017-2022, OokiDao. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0.
+ */
+
+pragma solidity >=0.5.0 <0.6.0;
+
+
+interface IWeth {
```

Comment:
we already have weth interface somewhere. lets reuse it.

Reply:
fixed.